coderd/provisionerdserver/provisionerdserver_test.go

```diff
@@ -1516,6 +1521,63 @@ func TestInsertWorkspaceResource(t *testing.T) {
 		got, err := agent.EnvironmentVariables.RawMessage.MarshalJSON()
 		require.NoError(t, err)
 		require.Equal(t, want, got)
+		require.ElementsMatch(t, []database.DisplayApp{
+			database.DisplayAppPortForwardingHelper,
+			database.DisplayAppSSHHelper,
+			database.DisplayAppVscode,
+		}, agent.DisplayApps)
+	})
+
+	t.Run("AllDefaultApps", func(t *testing.T) {
+		t.Parallel()
+		db := dbfake.New()
+		job := uuid.New()
+		err := insert(db, job, &sdkproto.Resource{
+			Name: "something",
+			Type: "aws_instance",
+			Agents: []*sdkproto.Agent{{
+				DisplayApps: &sdkproto.DisplayApps{
```

Comment:
I'm wondering if `DisplayApps` can refer to provisionersdk or a similar place to pull in these defaults.

Reply:
I added this but it doesn't actually apply to this test case since it's testing to ensure all possible values are translated correctly.